content/blog/2020-05-googlesheets4-0-2-0/index.Rmarkdown.orig

```diff
@@ -0,0 +1,186 @@
+---
+title: googlesheets4 0.2.0
+author: Jenny Bryan
+date: '2020-05-10'
+slug: googlesheets4-0-2-0
+categories:
+  - package
+tags:
+  - googlesheets4
+  - google
+photo:
+  author: Stephane YAICH
+  url: https://unsplash.com/photos/ufxd6aU0w9U
+---
+
+<!-- index.Rmarkdown is generated from index.Rmarkdown.orig -->
```

Comment:
I'm mildly concerned about this extra layer of indirection, because we should be getting this from the existing `.Rmarkdown` files when combined with the file touching hack in the readme. But it's not particularly important to resolve if you are ok managing the multiple levels of rendering that's now involved.

Reply:
I suspect we should leave the extra level of rendering. Because if (when, really) blogdown tries to render this on a machine that can't decrypt the token, they're going to get an even weirder failure than usual.

Reply:
But OK, I'll undo this and see how it goes.

Reply:
This is more a registration of my frustration with blogdown than an actionable insight 😬